packages/backend/src/peripherals/starkware/schema/PerpetualLiveL2TransactionResponse.ts

```diff
@@ -5,14 +5,19 @@ import { PerpetualL2Transaction } from './PerpetualBatchInfoResponse'
 const PerpetualLiveL2TransactionResponseTransactionInfo = z.object({
   tx: PerpetualL2Transaction,
   tx_id: z.number(),
+  parseError: z.undefined(),
```

Comment:
why is that z.undefined?

Reply:
Removed here and added as a separate discriminated union called ParsedL2TransactionInfo